### ISSUE TITLE
Would you consider adding type hints to your project?

### ISSUE THREAD

**Issue by Colin-b** [michaelkryukov/mongomock_motor#29]
And then package it as typed so that mypy can run checks on it?

**Comment by DavidKaub** [comment]
@michaelkryukov assuming I'd provide the types; Would you merge the PR?

**Comment by michaelkryukov** [comment]
@DavidKaub yes, I'll review the PR, and if the changes are beneficial and don't introduce any issues, I'll merge it. Thanks!
